src/librustdoc/clean/mod.rs

```diff
@@ -194,6 +195,18 @@ impl<'a, 'tcx, 'rcx> Clean<Crate> for visit_ast::RustdocVisitor<'a, 'tcx, 'rcx>
                     inner: PrimitiveItem(prim),
                 }
             }));
+            m.items.extend(keywords.iter().map(|&(def_id, ref kw, ref attrs)| {
```

Comment:
I wonder if you could use `into_iter` with this to prevent the clones below. `keywords` isn't really used outside of this, so there's no use keeping it around.

Reply:
Excellent idea!